change_log/v9_0_0/FE-1944-table-ajax-errors.yml

```
@@ -0,0 +1,2 @@
+Storybook Update: Fixed documentation 'info' and 'note' implementation to avoid duplication that was triggering not unique key console warning.
+Component Update: Fixed, totalRecords prop will now be accepted and passed as a Number not as a String.
```

Should really prefix this with the component it's relevant too.
```suggestion
Component Update: TableAjax - totalRecords prop will now be accepted and passed as a Number not as a String.
```